hwloc.spec

```diff
@@ -1,8 +1,8 @@
-### RPM external hwloc 2.7.1
-Source: https://download.open-mpi.org/release/%{n}/v2.7/%{n}-%{realversion}.tar.bz2
+### RPM external hwloc 2.8.0
+Source: https://download.open-mpi.org/release/%{n}/v2.8/%{n}-%{realversion}.tar.bz2
 
 BuildRequires: autotools
-Requires: cuda libpciaccess libxml2 numactl
+Requires: cuda libpciaccess libxml2 numactl rocm
```

Comment:
@fwyzard , `rocm` is only available for `x86_64`. So I guess this should be a conditional dependency i.e.
```
%ifarch x86_64
Requires: rocm
%endif
```

Reply:
Yes, and also the dependency on cuda should be confitional. I'll slowly make those changes.